gemma-web/src/main/config/log4j-dev.properties

```diff
@@ -69,6 +69,10 @@ log4j.logger.ubic.gemma.grid.javaspaces=INFO
 # Jersey
 log4j.logger.org.glassfish.jersey.internal.Errors=ERROR
 
+# Jawr
+log4j.logger.net.jawr.web.servlet.JawrRequestHandler=ERROR
+log4j.logger.net.jawr.web.servlet.JawrBinaryResourceRequestHandler=ERROR
```

Comment:
This should be a separate commit.

Reply:
It's fine to ignore thee warning in the dev config as JAWR is only complaining about the debug mode being used.